doc/api/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>API Documentation</title>
+  <meta charset="utf-8">
+  <style>
+    
+body {
```

Comment:
Can we move the CSS code to a different file? Same suggestion for the rest of the html files

Reply:
This files are auto-generated. I think these files should not be updated, wdyt?